package.json

```diff
@@ -32,6 +33,21 @@
     "@types/node": "^18.0.0",
     "@types/node-cron": "^3.0.1",
     "eslint": "^8.18.0",
+=======
+    "@discordjs/builders": "^0.13.0",
+    "@discordjs/rest": "^0.4.1",
+    "@discordjs/voice": "^0.14.0",
+    "@psibean/discord.js-pagination": "^4.0.5",
+    "discord-api-types": "^0.32.0",
+    "discord.js": "^14.7.1",
+    "mongoose": "^6.3.1",
+    "node-cron": "^3.0.0"
+  },
+  "devDependencies": {
+    "@types/node": "^17.0.27",
+    "@types/node-cron": "^3.0.6",
+    "eslint": "^8.3.0",
+>>>>>>> Stashed changes
```

Comment:
And here

Reply:
@Juilzze resolved this issue in latest commit.